discoverable-client/src/main/resources/application.yml

```diff
@@ -78,6 +78,9 @@ apiml:
                 enableUrlEncodedCharacters: true
                 gatewayPort: 10010
                 gatewayAuthEndpoint: /api/v1/gateway/auth
+                connectTimeout: 5684
+                readTimeout: 75437
```

Comment:
Is it possible to either remove these values or set them to something meaningful.